extensions/common/gcp/gcp-core/src/main/java/org/eclipse/edc/gcp/bigquery/BigQueryTarget.java

```diff
@@ -22,6 +22,10 @@ public TableId getTableId() {
         return TableId.of(project, dataset, table);
     }
 
+    public TableId getTableId(String tableName) {
```

Comment:
not directly related, but I think the `BigQueryTarget` class deserves some javadoc, esp. what it's intended purpose is.

Reply:
Done